### PR TITLE
popUp

### PR DESCRIPTION
**Pull request by IhorKalyta**

### CODE REVIEW

File: client/src/components/PopUp/PopUp.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import React from 'react';
+import Cropper from '../Cropper/Cropper';
+
+export default class PopUpModal extends React.Component<{}, { modal: boolean}> {
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: React.Component<IInitialModalState, IModalState>

create interface

File: client/src/components/PopUp/PopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    constructor(props: any) {
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: IInitialModalState?

File: client/src/components/PopUp/PopUp.tsx (hunk 1 above)
Comment (IhorKalyta) [reply client/src/components/PopUp/PopUp.tsx]: I have created two Interfaces.

File: client/src/components/PopUp/PopUp.tsx (hunk 2 above)
Comment (IhorKalyta) [reply client/src/components/PopUp/PopUp.tsx]: Fixed

File: client/src/components/PopUp/PopUp.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+interface IInitialModalState {
+    initialModalState: boolean;
+}
+
+export default class PopUpModal extends React.Component<IInitialModalState, IModalState> {
+    constructor(props: IInitialModalState) {
+        super(props);
+        this.state = {
+            modal: props.initialModalState,
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: it is enough - false
         ```modal: false``

File: client/src/components/PopUp/PopUp.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import Cropper from '../Cropper/Cropper';
+
+interface IModalState {
+    modal: boolean;
+}
+interface IInitialModalState {
+    initialModalState: boolean;
+}
+
+export default class PopUpModal extends React.Component<IInitialModalState, IModalState> {
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: no need IInitialModalState -> null

File: client/src/components/Profile/Profile.tsx [hunk 5]
```diff
@@ -102,7 +102,7 @@ export default class Profile extends React.Component<IFormProps> {
                         </div>
                     </div>
                 </div>
-                <Cropper/>
+                <PopUpModal initialModalState={false}/>
```
Comment (InnaKutsik) [inline client/src/components/Profile/Profile.tsx]: no need this props initialModalState

File: client/src/components/PopUp/PopUp.tsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import React from 'react';
+import Cropper from '../Cropper/Cropper';
+
+interface IModalState {
+    modal: boolean;
+}
+
+export default class PopUpModal extends React.Component<{}, IModalState> {
+    constructor(props: IModalState) {
```
Comment (InnaKutsik) [inline client/src/components/PopUp/PopUp.tsx]: {}

**Pull request merged**